client/js/lounge.js

```diff
@@ -420,9 +420,12 @@ $(function() {
 
 		// Remove the date-change marker we put at the top, because it may
 		// not actually be a date change now
-		var firstChild = $(chan).children().eq(0);
-		if (firstChild.attr("class") === "date-marker") {
-			firstChild.remove();
+		var children = $(chan).children();
+		if (children.eq(0).attr("class") === "date-marker") { // Check top most child
```

Comment:
Ah wait, can you use jquery's hasClass instead? Because this will only give oyu 1 of the classes, which means that it could create bugs in the future if the date marker ends up having more than 1 class on it.

https://api.jquery.com/hasclass/